backend/src/packages/users/user-details.model.ts

```diff
@@ -26,13 +26,10 @@ class UserDetailsModel extends AbstractModel {
         modelClass: UserModel,
         join: {
           from: composeDatabaseRelationPath<UserDetailsModel>(
-            DatabaseTableName.USER_DETAILS,
+            'user_details',
```

Comment:
Turn it back to use DatabaseTableName enum

Reply:
Sorry, didn't check after pull